politeiawww/config.go

```diff
@@ -552,8 +553,34 @@ func loadConfig() (*config, []string, error) {
 	cfg.HTTPSCert = cleanAndExpandPath(cfg.HTTPSCert)
 	cfg.RPCCert = cleanAndExpandPath(cfg.RPCCert)
 
-	cfg.CacheCertDir = cleanAndExpandPath(cfg.CacheCertDir)
+	// Validate cache options.
+	switch {
+	case cfg.CacheHost == "":
+		return nil, nil, fmt.Errorf("the cachehost param is required")
+	case cfg.CacheRootCert == "":
+		return nil, nil, fmt.Errorf("the cacherootcert param is required")
+	case cfg.CacheCert == "":
+		return nil, nil, fmt.Errorf("the cachecert param is required")
+	case cfg.CacheKey == "":
+		return nil, nil, fmt.Errorf("the cachekey param is required")
+	}
+
 	cfg.CacheRootCert = cleanAndExpandPath(cfg.CacheRootCert)
+	cfg.CacheCert = cleanAndExpandPath(cfg.CacheCert)
+	cfg.CacheKey = cleanAndExpandPath(cfg.CacheKey)
+
```

Comment:
A check that tls.LoadX509KeyPair() does not fail is needed.  Then two os.Stat() checks below can be removed.  A better check than just existing on the root cert is needed as well.

Reply:
Fixed.  The key pair is checked using tls.LoadX509KeyPair() and the root cert is checked using x509.ParseCertificate().